java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -672,4 +673,79 @@ CompletableFuture<Object[][]> xpending(
             StreamRange end,
             long count,
             StreamPendingOptions options);
+
+    /**
+     * Changes the ownership of a pending message.
+     *
+     * @see <a href="https://valkey.io/commands/xclaim/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param group The consumer group name.
+     * @param consumer The group consumer.
+     * @param minIdleTime The minimum idle time for the message to be claimed.
+     * @param ids An array of entry ids.
+     * @return An <code>array</code> of message entries with the format <code>
+     *     [[id, ["entry", "data"]], ...]</code> that are claimed by the consumer.
+     * @example
+     *     <pre>
+     * </pre>
+     */
+    CompletableFuture<Map<String, String[]>> xclaim(
+            String key, String group, String consumer, long minIdleTime, String[] ids);
+
+    /**
+     * Changes the ownership of a pending message.
+     *
+     * @see <a href="https://valkey.io/commands/xclaim/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param group The consumer group name.
+     * @param consumer The group consumer.
+     * @param minIdleTime The minimum idle time for the message to be claimed.
+     * @param ids An array of entry ids.
+     * @param options Stream claim options {@link StreamClaimOptions}.
+     * @return An <code>array</code> of message entries with the format <code>
+     *     [[id, ["entry", "data"]], ...]</code> that are claimed by the consumer.
+     */
+    CompletableFuture<Map<String, String[]>> xclaim(
+            String key,
+            String group,
+            String consumer,
+            long minIdleTime,
+            String[] ids,
+            StreamClaimOptions options);
+
+    /**
+     * Changes the ownership of a pending message. This command uses the JUSTID optional argument to
```

Comment:
It is optinal for command in Redis API, but not in this very function.
Maybe reword if you think it is needed:
> Changes ... This function makes redis command to return only message/entry ID, since it uses JUSTID command/Redis API argument.

Reply:
Wrap JUSTID with `code`